### ISSUE TITLE
[openhabcloud] openHAB Cloud Connector connection error

### ISSUE THREAD

**Issue by rangertwo** [openhab/openhab2-addons#3341]
openHAB Cloud Connector, Myopenhab connection to dashboard is not working.

In general the connection is working, openhab is showen as online and itemupdates are working in 
Working: 
https://myopenhab.org/events
https://myopenhab.org/items
https://myopenhab.org/notifications

Not working:
https://home.myopenhab.org/start/index

## Current Behavior
18:33:08.792 [ERROR] [.io.openhabcloud.internal.CloudClient] - Verbindungsaufbau abgelehnt
18:33:08.798 [WARN ] [.io.openhabcloud.internal.CloudClient] - Jetty request 1413185 failed: Verbindungsaufbau abgelehnt
18:33:08.817 [WARN ] [.io.openhabcloud.internal.CloudClient] - Verbindungsaufbau abgelehnt
18:33:08.822 [WARN ] [.io.openhabcloud.internal.CloudClient] - Verbindungsaufbau abgelehnt

## Your Environment
openHAB 2.2.0 Release Build
openhab-addons-2.2.0
java version "1.8.0_162"
Java(TM) SE Runtime Environment (build 1.8.0_162-b12)
Java HotSpot(TM) Client VM (build 25.162-b12, mixed mode)
PRETTY_NAME="Raspbian GNU/Linux 8 (jessie)"
NAME="Raspbian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"

**Comment by wborn** [comment]
Do you still have this issue @rangertwo?

**Comment by witek1308** [comment]
I have the same problem and it drives me nuts. I have a feeling that I'm missing some important configuration as there is no magic in it. Quickly:
- notifications, events, items work fine
- dashboard https://home.myopenhab.org/start/index
- when I try to connect with Android app on the phone to remote link, I'm getting the same error
- when I try to integrate Google Home with the OpenHAB I'm getting the same error
Obviously in the OH Cloud Connector I allow for remote access (at least this is shown in the GUI).
I reinstalled OH Cloud Connector, recreated account 2x in myopenhab.org cloud. Results are the same
In other words any access from outside is refused. My gutfeeling is that it's desired behavior until some configuration is added/changed but I'm not sure. In such a case however I'd have found some hints on the internet whereas all guides and posts suggest that integration to the cloud is straightforward with no additional file based config. I'm looking for an advice where to start looking to fix it.

**Comment by witek1308** [comment]
I've found the problem (it is described in OpenHAB community forum). Simply the other day I asked OH to stick to a specific network interface which disabled listening on localhost (I rather expected it disables all external interfaces only but keeps lo) that's why it took me some time to find out this very simple thing as I was rather expecting complex problem and haven't checked basics... :)

**Comment by J-N-K** [comment]
seems to be a condifuration issue.
